src/pages/about-data/visualization-guide/index.js

```diff
@@ -280,5 +266,13 @@ export const query = graphql`
         }
       }
     }
+    file(relativePath: { regex: "/cdc-comparison-chart.png/" }) {
+      relativePath
```

Comment:
I think this query is better suited to live within the component as it's currently the only "chart" whose data is fetched within this page (and not in it's own sub component).